### DISCUSSION TITLE
Search for keyword in query code (not name)

### DISCUSSION THREAD

**Discussion by JrgMyr** [pawelsalawa/sqlitestudio#4067] [Ideas]
Dear all.
Is there a known way to select queries/views by searching through their content?
[Tables/queries/views can be filtered by name. That's not meant.]

Examples:
- Show me all queries that contain a certain field name.
- Show me all views that apply a certain string value in a WHERE clause.

Assuming this is not the case, may I suggest this as a new feature?

**Comment by pawelsalawa** [comment]
It sounds like ability to finding all references to a particular field, or table, or index, etc. There is no such feature right now and I think it's a neat one to have. Yes, I'll create improvement proposal from this.

**Reply by pawelsalawa** [reply]
Created #4068

**Reply by JrgMyr** [reply]
Would that allow to search for arbitrary text values or only for objects that exist in the database?
Say, I want to search for a certain query pattern or a string literal. Would that be covered?

**Reply by pawelsalawa** [reply]
I guess it could. I'll extend the description.
